source/slang/hlsl.meta.slang

```diff
@@ -9751,9 +9752,11 @@ bool __reportIntersection(float tHit, uint hitKind)
 {
     __target_switch
     {
-    case _GL_EXT_ray_tracing: __intrinsic_asm "reportIntersectionEXT";
+    case _GL_EXT_ray_tracing:
+    case glsl: __intrinsic_asm "reportIntersectionEXT";
```

Comment:
You shouldn't need `case glsl` if the requirements are properly marked in glsl.meta.slang